shared/chat/conversation/messages/wrapper/wrapper-timestamp.js

```diff
@@ -0,0 +1,36 @@
+// @flow
```

Comment:
This folder is getting kinda crowded. Do you think this + `wrapper-timestamp-container` could move into a subfolder?